meta-box/meta-box-template.php

```diff
@@ -22,39 +22,39 @@
 <p>
 	<b>
 		<span class="dashicons dashicons-yes"></span>
-		This post will be available as an Instant Article.
+		This post will be available as Instant Article.
 	</b>
 </p>
 <hr>
 <?php elseif ( ! $instant_articles_should_submit_post_filter ) : ?>
 <p>
 	<b>
 		<span class="dashicons dashicons-no-alt"></span>
-		This post will not be submitted to Instant Articles due to a rule created in your site.
+		This post will not be available as Instant Article due to a rule created in your site.
 	</b>
 </p>
 <hr>
 <?php elseif ( ! $published ) : ?>
 <p>
 	<b>
 		<span class="dashicons dashicons-media-document"></span>
-		This post will be submitted to Instant Articles once it is published.
+		This post will be available as Instant Article once it is published.
```

Comment:
Here too:

"This post will be available as Instant Article once it is published and shared on Facebook"